nixos/community.tt

```diff
@@ -84,7 +84,7 @@ you:</p>
 <h3>Mailing list</h3>
 
 <p>NixOS development is coordinated on the <a
-href="[%root%]nixos/support.html#mailing-list"><tt>nix-devel</tt>
+href="[%root%]nixos/support.html#mailing-list"><tt>Discourse</tt>
```

Comment:
Is this actually true? I think it is more coordinated on GitHub and IRC if anything.

Reply:
fixed in 76a4f50159e3ae9694692e3b377a4d0cc5ccfe77